config/api.js

```diff
@@ -7,7 +7,8 @@ import { convertToBackendFilters } from "~/components/UnifiedDocFeed/utils/conve
 export const apiRoot = {
   production: "backend.researchhub.com",
   staging: "staging-backend.researchhub.com",
-  dev: "localhost:8000",
+  // dev: "localhost:8000",
+  dev: "127.0.0.1:8000" /* localhost */,
```

Comment:
Can we keep localhost? There is some functionality that utilizes "localhost" as the HOST